.gitignore

```diff
@@ -39,6 +39,7 @@ testem.log
 Thumbs.db
 
 .nx/cache
+/.pnpm-store
```

Comment:
Do we need this?

Reply:
I don't know why this happens, but some times `pnpm` in the dev container is added to the project directory.
It's strange. But it wouldn't be out of place.